slides/snippets/snippets.js

```diff
@@ -122,12 +122,6 @@ function createTextboxWithText(presentationId, pageId, callback) {
 function createImage(presentationId, pageId, imageFilePath, imageMimetype, callback) {
```

Comment:
As per other samples, remove file path and mime type parameters.

Reply:
Done.

Reply:
Done.